src/transactions/Transaction/cashTransaction.js

```diff
@@ -0,0 +1,8 @@
+import Transaction from './transaction'
+import { ICashTransaction } from '../../../index.d'
```

Comment:
What you had in here before was correct.

This is not a TypeScript project, so you won't be able to use `.ts` files. This is just a vanilla JS file, all the TS stuff that TS projects need in order to use this project should be contained within `index.d.ts`.